README.md

```diff
@@ -34,6 +34,8 @@ This app officially supports GitHub.com (which includes our GitHub Enterprise cl
 
 <p align="center"><img width="450" alt="auth" src="https://user-images.githubusercontent.com/3877742/36522927-f1d596b6-1753-11e8-9f85-2495e657b16b.png"></p>
 
+**Note:** At this time this integration only works with public Slack channels. While a private Slack channel can subscribe to a repository, it will not post notifications.
```

Comment:
This is not the case. App works for the private channels too. To get it working app must be invited to the private channel.
Also thing to note is that, when installing the app, it should not be installed only on a specific channel. If it's installed for all channels and app is invited in the channel, it must work.

Reply:
Then it sounds like the best solution is to update this note and say "app will not work on private channels unless app has access to all channels in initial integration?"

Reply:
This point is already mentioned in the next line. (line no 39) :)